src/Razor/src/Microsoft.CodeAnalysis.Remote.Razor/RemoteTagHelperProviderService.cs

```diff
@@ -15,14 +15,20 @@ namespace Microsoft.CodeAnalysis.Remote.Razor
 {
     internal sealed class RemoteTagHelperProviderService : RazorServiceBase, IRemoteTagHelperProviderService
     {
+        private readonly RemoteTagHelperDeltaProvider _tagHelperDeltaProvider;
+
         internal RemoteTagHelperProviderService(IServiceBroker serviceBroker)
             : base(serviceBroker)
         {
+            _tagHelperDeltaProvider = new RemoteTagHelperDeltaProvider();
         }
 
         public ValueTask<TagHelperResolutionResult> GetTagHelpersAsync(RazorPinnedSolutionInfoWrapper solutionInfo, ProjectSnapshotHandle projectHandle, string factoryTypeName, CancellationToken cancellationToken = default)
             => RazorBrokeredServiceImplementation.RunServiceAsync(cancellationToken => GetTagHelpersCoreAsync(solutionInfo, projectHandle, factoryTypeName, cancellationToken), cancellationToken);
 
+        public ValueTask<TagHelperDeltaResult> GetTagHelpersDeltaAsync(RazorPinnedSolutionInfoWrapper solutionInfo, ProjectSnapshotHandle projectHandle, string factoryTypeName, int lastResultId, CancellationToken cancellationToken)
```

Comment:
This is the actual entry point on the ServiceHub side that's invoked in response to the `OOPTagHelperResolver` asking for deltas